setup.cfg

```
@@ -64,7 +64,7 @@ testing =
   # upstream
   pytest >= 7
   pytest-checkdocs >= 2.4
-  pytest-flake8
+  pytest-flake8 >= 1.0
```

```suggestion
  pytest-flake8 >= 1.0.7
```
this should match the value in tests/requirements.txt